### PR TITLE
Css styling

### PR DESCRIPTION
**Pull request by skwong25**
Bless this mess

### CODE REVIEW

**Comment by skwong25** [comment]
Aware this is now showing errors with npm test!

File: src/Title.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+
+export default function Title(props) {
```
Comment (fractalwrench) [inline src/Title.js:5]: This is a good component :) it's also possible (and more conventional) to specify the title in the `<head>` element of your `index.html`.

File: src/SubmitSearch.js [hunk 2]
```diff
@@ -4,23 +4,36 @@
...
+                    </Paper>
+                </Grid>
             </div>
         ) 
     } else {
         return (
             <div>
                 <br/>
-                <button className="Refresh" onClick={()=>window.location.reload(false)}>Refresh</button>
+                <Button color="primar" className="Refresh" onClick={()=>window.location.reload(false)}>Refresh</Button>
```
Comment (fractalwrench) [inline src/SubmitSearch.js:32]: primar -> primary should fix the colour

**Comment by skwong25** [comment]
> Looking much better! A few general comments to help you with the right direction, which you have probably thought of somewhat already:
> 
>     * .DS_Store should be added to .gitignore
> 
>     * It'd be nice to have the main container expand to say around 50% of the screen width (then fit 100% of the screen on mobile with a [responsive design](https://www.w3schools.com/html/html_responsive.asp))
> 
>     * It would be good to have some padding around elements
> 
>     * Maybe the page could use some sort of banner image showing movies to make it look lively? A bit like the 'Welcome to React' banner. This could probably be an extended project after we've done to the CSS

**Comment by skwong25** [comment]
Response to your previous comments: 
- .DS_Store has been removed, thanks for your assistance. 
- When viewed at 170%, the content fills the entire page to mimic view on a small mobile screen. The spacing has now been sorted. 
- The fuschia colour scheme responds to your previous request (see above) to make the page look more 'lively'. I hope it has met your expectations and more.

**Pull request merged**